### ISSUE TITLE
ModuleNotFoundError: No module named 'cometblue'

### ISSUE THREAD

**Issue by giddyhup** [zero-udo/eurotronic-cometblue#4]
Thanks for the update. Regrettably, the original module is hiding now (although it should be in the search path):

```
pi@berry ~> pip3 install eurotronic-cometblue
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Requirement already satisfied: eurotronic-cometblue in /usr/local/lib/python3.7/dist-packages (1.0.2)
Requirement already satisfied: pygatt[gatttool]==4.0.5 in ./.local/lib/python3.7/site-packages (from eurotronic-cometblue) (4.0.5)
Requirement already satisfied: pyserial in /usr/lib/python3/dist-packages (from pygatt[gatttool]==4.0.5->eurotronic-cometblue) (3.4)
Requirement already satisfied: enum-compat in ./.local/lib/python3.7/site-packages (from pygatt[gatttool]==4.0.5->eurotronic-cometblue) (0.0.3)
Requirement already satisfied: pexpect; extra == "gatttool" in /usr/lib/python3/dist-packages (from pygatt[gatttool]==4.0.5->eurotronic-cometblue) (4.6.0)
pi@berry ~> python3
Python 3.7.3 (default, Jan 22 2021, 20:04:44)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from cometblue import CometBlue, Weekday
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'cometblue'
>>>
pi@berry ~> sudo python3
Python 3.7.3 (default, Jan 22 2021, 20:04:44)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from cometblue import CometBlue
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'cometblue'
>>>
```

**Comment by zero-udo** [comment]
For whatever reason the module was renamed during build to 'CometBlue' instead of 'cometblue'.
A fixed version (1.0.3) was published.

To avoid future problems i would advise that you remove the current module and reinstall instead of (just) updating.
So:
`pip uninstall eurotronic-cometblue`
to remove the wrongfully created 'CometBlue' folders in site-packages - then afterwards
`pip install eurotronic-cometblue` 
which should install version 1.0.3 with the correct naming.

**Comment by giddyhup** [comment]
Thanks. It is working now. (Well, not really. Bluetooth stops working on my Pi 4 after a while but that is unrelated to your solution.)
